File: NativeDelegateTest/NativeDelegateTest/MyDelegate.m
//
//  MyDelegate.m
//  NativeDelegateTest
//
//  Created by Martin Baulig on 10/29/12.
//  Copyright (c) 2012 Martin Baulig. All rights reserved.
//

#import "MyDelegate.h"

@implementation MyDelegate

- (void)textViewDidChangeSelection:(NSNotification *)aNotification {
	printf("TEXT VIEW DID CHANGE SELECTION!\n");
}

- (void)textDidChange:(NSNotification *)aNotification {
	printf ("TEXT DID CHANGE!\n");
}

@end

File: DelegateTest/MyDelegate.cs
using System;
using MonoMac.AppKit;
using MonoMac.Foundation;

namespace DelegateTest
{
	public class MyDelegate : NSTextViewDelegate
	{
		public override void DidChangeSelection (NSNotification notification)
		{
			Console.WriteLine ("DID CHANGE SELECTION!");
		}

		public override void TextDidChange (NSNotification notification)
		{
			Console.WriteLine ("TEXT DID CHANGE!");
		}
	}
}

